src/linear_algebra/affine_space/affine_equiv.lean

```diff
@@ -340,7 +340,8 @@ include V
 /-- Fixing a point in affine space, homothety about this point gives a group homomorphism from (the
 centre of) the units of the scalars into the group of affine equivalences. -/
 def homothety_units_mul_hom (p : P) : Rˣ →* P ≃ᵃ[R] P :=
-equiv_units_affine_map.symm.to_monoid_hom.comp $ units.map (affine_map.homothety_hom p)
+monoid_hom.comp (mul_equiv.to_monoid_hom equiv_units_affine_map.symm)
```

Comment:
The following works:
```
def homothety_units_mul_hom (p : P) : Rˣ →* P ≃ᵃ[R] P :=
begin
  have f := (@equiv_units_affine_map R P _ _ _ _ _).symm,
  exact (f.to_monoid_hom).comp (units.map (affine_map.homothety_hom p)),
end
```

but just copying `f` into the expression gives an error:
```
/-- Fixing a point in affine space, homothety about this point gives a group homomorphism from (the
centre of) the units of the scalars into the group of affine equivalences. -/
def homothety_units_mul_hom (p : P) : Rˣ →* P ≃ᵃ[R] P :=
begin
  exact (((@equiv_units_affine_map R P _ _ _ _ _).symm).to_monoid_hom).comp (units.map (affine_map.homothety_hom p)),
end
```
the error is:
```
type mismatch at application
  equiv_units_affine_map.symm.to_monoid_hom
term
  equiv_units_affine_map.symm
has type
  (P →ᵃ[R] P)ˣ ≃* (P ≃ᵃ[R] P) : Type (max (max u_11 u_12) u_12 u_11)
but is expected to have type
  ?m_1 ≃* ?m_2 : Type (max ? ?)
```

What is going on? :(